.github/workflows/Image.yaml

```diff
@@ -27,6 +27,7 @@ on:
 env:
   YQ_VERSION: "v4.32.2"
   YQ_BINARY: yq_linux_amd64
```

Comment:
Side point that's not very important but saves these details:

- Generally I think `snap install yq` is likely to be the right route
- I **think** `yq` is already installed on the Github Actions runners ;-)

Reply:
For the second point, I think it would be better if we install the software ourselves to ensure we have the correct version of 'yq'.

Reply:
Oh, indeed it already has `yq` installed (v4.35.1). Thanks for the tip. Being addressed in https://github.com/canonical/oci-factory/pull/56